docs/supported-tools.md

```diff
@@ -275,6 +276,14 @@ The `filters` property expects a string consisting in a comma-separated list of
 If your project mixes Java and Kotlin code, you don't need to have an exclusion in place for all `*.java` files. Detekt itself only looks for
 `*.kt` files, so no further configuration is required.
 
+## Android Lint
+[Android Lint](https://developer.android.com/studio/write/lint.html) is a code scanning tool with focus on Android. Besides code it also analyses resources
+and configuration files.
+
+### Configure Android Lint
+Lint is configured through the `lintOptions` closure. It supports all [official](https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.internal.dsl.LintOptions.html)
+properties except `abortOnError`, `htmlReport` and `xmlReport`. These are hardcoded so lint won't break the build by it's own and always generates reports.
+
```

Comment:
I would add a warning that before the Android Gradle plugin v3.1 (not sure what exact canary version), Lint cannot find issues in Kotlin files. It's not _our_ fault, but I imagine some may come reporting that as an issue.